README.md

```diff
@@ -54,7 +53,9 @@ https://oneapi.io/ for more information.
 ### AI - Machine Learning
 
 * [DQRM](https://anonymous.4open.science/r/Deep_Quantized_Recommendation_Model_DQRM-6B4D) - Deep Quantized Recommendation Model (DQRM) is a recommendation framework that is small, powerful in inference, and efficient to train.
+* [ort](https://github.com/pykeio/ort) - ort is an (unofficial) ONNX Runtime 1.15 wrapper for Rust based on the now inactive onnxruntime-rs. ONNX Runtime accelerates ML inference on both CPU & GPU.
```

Comment:
Is a project that has the option to link against the oneAPI libraries worth of considering?

Reply:
Absolutely.